frontend/pages/project-developer/[id]/index.tsx

```diff
@@ -1,108 +1,126 @@
-// import { useRouter } from 'next/router';
-import { useIntl } from 'react-intl';
+import { useMemo } from 'react';
+
 import { FormattedMessage } from 'react-intl';
 
-import { chunk } from 'lodash-es';
+import { decycle } from 'cycle';
+import { chunk, groupBy } from 'lodash-es';
 
 import { loadI18nMessages } from 'helpers/i18n';
 
 import ProfileHeader from 'containers/profile-header';
 import ProjectCard from 'containers/project-card';
-import { SocialType } from 'containers/social-contact/website-social-contact';
+import { SOCIAL_DATA } from 'containers/social-contact/constants';
 import TagsGrid, { TagsGridRowType } from 'containers/tags-grid';
 
 import Carousel, { Slide } from 'components/carousel';
 import Head from 'components/head';
 import LayoutContainer from 'components/layout-container';
+import { EnumTypes, Paths } from 'enums';
 import { StaticPageLayoutProps } from 'layouts/static-page';
 import { PageComponent } from 'types';
+import { CategoryType } from 'types/category';
+import { GroupedEnums as GroupedEnumsType } from 'types/enums';
+import { ProjectDeveloper as ProjectDeveloperType } from 'types/projectDeveloper';
+
+import { getEnums } from 'services/enums/enumService';
+import { getProjectDeveloper } from 'services/project-developers/projectDevelopersService';
+
+export const getServerSideProps = async ({ params: { id }, locale }) => {
+  let projectDeveloper;
+
+  // If getting the project developer fails, it's most likely because the record has
+  // not been found. Let's return a 404. Anything else will trigger a 500 by default.
+  try {
+    ({ data: projectDeveloper } = await getProjectDeveloper(id, { includes: 'projects' }));
```

Comment:
I guess the list of projects is paginated. How do you make sure we display all of them in the page?

Reply:
The list of projects comes from the `/api/v1/project_developers/{id}` endpoint via `includes=projects`. It is not paginated. 

The current "pagination" is being done via the `Carousel` component and its slides. See [here](https://github.com/Vizzuality/heco-invest/blob/develop/frontend/pages/project-developer/%5Bid%5D/index.tsx#L139).